css/src/tokens/ratios.scss

```diff
@@ -0,0 +1,10 @@
+/**
+ * @sass-export-section="ratios"
+ */
+
+$ratios: (
```

Comment:
While we're adding these atomics, we should probably port over these ratios found in `packages\styles\src\elements\_image.scss` as well for full compatibility, so we can fully deprecate the older ratio utility classes:

- 5 / 4
- 3 / 2
- 2 / 1
- 3 / 1
- 4 / 5
- 3 / 4
- 2 / 3
- 1 / 2
- 1 / 3

I _think_ that's it.